com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/MtaConfigurationPurger.java

```diff
@@ -148,6 +148,7 @@ private ConfigurationEntry toConfigurationEntry(MtaMetadata metadata, String pro
                                       null,
                                       null,
                                       null,
+                                      null,
```

Comment:
Add builder in ConfigurationEntry class or at least another constructs, passing multiple null arguments is awful.

Reply:
added constructor as I don't wan't to add more refactorings